### PR TITLE
Document Kibana Space ID setting

### PR DESCRIPTION
**Pull request by ycombinator**
## What does this PR do?

Documents the `space.id` setting available when setting up Kibana assets.

## Why is it important?

So users can understand how and when to use this setting.

## Checklist

- [ ] ~My code follows the style guidelines of this project~
- [ ] ~I have commented my code, particularly in hard-to-understand areas~
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [ ] ~I have added tests that prove my fix is effective or that my feature works~

### CODE REVIEW

File: libbeat/docs/shared-kibana-config.asciidoc [hunk 1]
```diff
@@ -85,6 +85,13 @@ An HTTP path prefix that is prepended to the HTTP API calls. This is useful for
 the cases where Kibana listens behind an HTTP reverse proxy that exports the API
 under a custom prefix.
 
+[float]
+[[kibana-space-id-option]]
+==== `setup.kibana.space.id`
+
+The Kibana Space ID to use. If specified, {beatname_uc} will load Kibana assets
+into this Kibana Space. Omit this option to use the Default Kibana Space.
```
Comment (dedemorton) [inline libbeat/docs/shared-kibana-config.asciidoc]:
```suggestion
The Kibana space ID to use. If specified, {beatname_uc} loads Kibana assets
into this Kibana space. Omit this option to use the default space.
```

Comment (dedemorton) [reply libbeat/docs/shared-kibana-config.asciidoc]: My edits are based on the capitalization/style conventions used in the Kibana topic about spaces: https://www.elastic.co/guide/en/kibana/7.6/xpack-spaces.html

Do you think it's worth adding a link to the docs about Kibana spaces? If so, you might want to make Kibana space an active link: {kibana-ref}/xpack-spaces.html[Kibana space]

File: auditbeat/auditbeat.reference.yml [hunk 2]
```diff
@@ -1133,6 +1133,9 @@ setup.kibana:
   # Optional HTTP path
   #path: ""
 
+  # Optional Space ID.
```
Comment (dedemorton) [inline auditbeat/auditbeat.reference.yml]:
```suggestion
  # Optional Kibana space ID.
```

Comment (dedemorton) [reply auditbeat/auditbeat.reference.yml]: I think it's worth qualifying here just like you do in the ref docs. Make this change globally.

**Comment by ycombinator** [comment]
@dedemorton I've addressed all your feedback. Please re-review when you get a chance. Thanks!

**Comment by ycombinator** [comment]
Travis CI is green and Jenkins CI failures are unrelated. Merging.

**Comment by ycombinator** [comment]
Travis CI is green and Jenkins CI failures are unrelated. Merging.

**Pull request merged**